### PR TITLE
19041 Created constants (enum) for resource phrases

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#19041

*Description of changes:*
- app version = 5.8.9
- created constants (enum) for common resource phrases
- updated misc resource files to use constants

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by severinbeauvais** [comment]
All, what do you think about this change -- worth doing or too abstract?

File: src/resources/ResourcePhrases.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/* eslint-disable max-len */
```
Comment (severinbeauvais) [inline src/resources/ResourcePhrases.ts:1]: I thought I'd have the strings all on one line (no breaks). Thoughts?

**Review by JazzarKarim — APPROVED** [review]
I like it 😃

File: src/resources/ResourcePhrases.ts (hunk 1 above)
Comment (JazzarKarim) [reply src/resources/ResourcePhrases.ts:1]: 👍

**Review by jamespaologarcia — APPROVED** [review]
Beautiful. LGTM

**Review by ketaki-deodhar — APPROVED** [review]
Wort it! :)

**Pull request merged**